configure/cpack_linux.cmake

```diff
@@ -0,0 +1,29 @@
+find_program(DPKG_CMD dpkg)
+execute_process(COMMAND ${DPKG_CMD} --print-architecture
+        OUTPUT_VARIABLE CPACK_DEBIAN_PACKAGE_ARCHITECTURE OUTPUT_STRIP_TRAILING_WHITESPACE)
+
+set(CPACK_GENERATOR "DEB")
+if(CMAKE_BUILD_TYPE STREQUAL "DEBUG")
+    set(CPACK_PACKAGE_NAME libtango9lts-dev)
+    set(CPACK_DEBIAN_PACKAGE_DEPENDS "libcos4-dev, libzmq5-dev | libzmq3-dev")
+    set(CPACK_DEBIAN_PACKAGE_CONFLICTS "libtango7-dev, libtango8-dev, libtango9-dev, libtango-dev")
+elseif(CMAKE_BUILD_TYPE STREQUAL "RELEASE")
+    set(CPACK_PACKAGE_NAME libtango9lts)
+    set(CPACK_DEBIAN_PACKAGE_DEPENDS "libcos4, libzmq5 | libzmq3")
+    set(CPACK_DEBIAN_PACKAGE_CONFLICTS "libtango7, libtango8, libtango9")
```

Comment:
In release mode, it looks like there is no conflict with libtango7 and libtango8 since the library files installed by these packages have different names (E.g.: /usr/lib/x86_64-linux-gnu/libtango.so.8. See https://packages.debian.org/jessie/amd64/libtango8/filelist).
This would not be true for the development packages since there might be a conflict with the installed include files(?), so in DEBUG CMake mode, I would keep the conflicts with libtango7-dev and libtango8-dev.